aten/src/ATen/cuda/CUDAGraph.cpp

```diff
@@ -159,10 +159,14 @@ void CUDAGraph::capture_end() {
   AT_CUDA_CHECK(cudaDriverGetVersion(&version));
   if (version < 11040) {
 #endif
+#if CUDA_VERSION >= 12000
```

Comment:
this would conflict with #91118? I'm fine with landing whatever's fastest to unblock, but let's please coordinate in #91122

Reply:
ah ok I wasn't aware of https://github.com/pytorch/pytorch/pull/91118. In this case let's land that one